### PR TITLE
add stars

### PR DESCRIPTION
**Pull request by 6houston6**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars__0">
```
Comment (polosanya) [inline src/index.html:11]:
```suggestion
    <div class="stars stars--0">
```
It's a modifier, not a block element

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
```
Comment (polosanya) [inline src/style.css:2]: Don't reset margins using * selector. It applies to all the elements, which is not good for performance

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+body {
+  padding: 32px 22px ;
```
Comment (polosanya) [inline src/style.css:6]: Do you really need these paddings?

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.stars__star:hover {
+  border: 1px rgb(247, 161, 80) solid;
+  border-radius: 6px;
+}
+
+.stars__star:active {
+  border: 1px rgb(80, 186, 247) solid;
+}
```
Comment (polosanya) [inline src/style.css:39]: It's cool that you implemented these styles. but there is nothing about this in task description

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+
+}
```
Comment (polosanya) [inline src/style.css:22]:
```suggestion
  margin-right: 4px;
}
```
I'm not sure that last stars should have this margin too
image

**Review by polosanya — CHANGES_REQUESTED** [review]
Pay attention to the tests. Let's try to delete the backstop_data folder and run npm i and tests again after changes.
Feel free to ask for help in fe_chat if you have any difficulties)

**Comment by 6houston6** [comment]
https://github.com/mate-academy/layout_stars/pull/2602#issue-1599737768
https://6houston6.github.io/layout_stars/report/html_report/

**Review by anstsot — APPROVED** [review]
Great job!

**Pull request closed without merging**